### ISSUE TITLE
GH Actions workflow tests-robotframework-lsp.yml should always archive the test results

### ISSUE THREAD

**Issue by otemek** [robocorp/robotframework-lsp#1036]
**Is your feature request related to a problem? Please describe.**
For now, test outputs are only archived when all the previous steps are executed correctly. It would be more concise if such execution results were archived regardless of the test results. Right now if there are any errors in the workflow I have to go through the errors in the browser github actions tab/ step window and wait until all of the lines load (sometimes it takes so long) because e.g. ~40k log lines to load :)

**Describe the solution you'd like**
Run [this step](https://github.com/robocorp/robotframework-lsp/blob/8a119f31f74e2dfe662452023fe085968ea3c1b2/.github/workflows/tests-robotframework-lsp.yml#L111) always. No matter what test results would be

**Describe alternatives you've considered**
Wait until all the log lines from the execution will load in a browser.

**Comment by otemek** [comment]
PR proposed [#1037](https://github.com/robocorp/robotframework-lsp/pull/1037)

**Comment by fabioz** [comment]
Agreed, would you like to do a PR (see: https://github.com/robocorp/robotframework-lsp/issues/1051)
